src/main/resources/db/migration/v0.0.1/V20190626114900__add_demo_data.sql

```diff
@@ -15,4 +15,36 @@ VALUES
+	(3, 0, '2017-05-01 08:35:25', '2019-04-16 08:35:25', 1, 3, '[{\"number\":1,\"text\":\"What is your housing situation today?\",\"answers\":[\"I have housing\",\"I do not have housing (staying with others, in a hotel, in a shelter, living outside on the street, on a beach, in a car, or in a park)\",\"I choose not to answer this question\"],\"answer\":\"I have housing\"},{\"number\":2,\"text\":\"Are you worried about losing your housing?\",\"answers\":[\"Yes\",\"No\",\"I choose not to answer this question\"],\"answer\":\"No\"}]'),
+	(4, 0, '2017-05-01 08:35:44', '2019-04-16 08:35:44', 1, 4, '[{\"number\":1,\"text\":\"In the past year, have you or any family members you live with been unable to get/afford any housing utilities (electricity, gas, etc) when it was really needed?\",\"answers\":[\"Yes\",\"No\",\"I choose not to answer this question\"],\"answer\":\"No\"}]'),
+	(5, 0, '2017-05-01 08:35:52', '2019-04-16 08:35:52', 1, 5, '[{\"number\":1,\"text\":\"Do you feel physically and emotionally safe where you currently live?\",\"answers\":[\"Yes\",\"No\",\"Unsure\",\"I choose not to answer this question\"],\"answer\":\"Yes\"},{\"number\":2,\"text\":\"In the past year, have you been afraid of your partner or ex-partner?\",\"answers\":[\"Yes\",\"No\",\"Unsure\",\"I have not had a partner in the past year\",\"I choose not to answer this question\"],\"answer\":\"No\"}]'),
+	(6, 0, '2017-05-01 08:38:56', '2019-04-16 08:38:56', 1, 6, '[{\"number\":1,\"text\":\"REALM-SF Score\",\"answers\":[\"0\",\"1\",\"2\",\"3\",\"4\",\"5\",\"6\",\"7\"],\"answer\":\"7\"}]'),
+	(7, 0, '2017-05-01 08:39:34', '2019-04-16 08:39:34', 1, 7, '[{\"number\":1,\"text\":\"What is your current work situation?\",\"answers\":[\"Unemployed\",\"Part-time or temporary work\",\"Full-time work\",\"Otherwise unemployed but not seeking work (ex: student, retired, disabled, unpaid primary care giver)\",\"I choose not to answer this question\"],\"answer\":\"Full-time work\"},{\"number\":2,\"text\":\"What is your main insurance?\",\"answers\":[\"None/uninsured\",\"Medicaid\",\"CHIP Medicaid\",\"Medicare\",\"Other public insurance (not CHIP)\",\"Other Public Insurance (CHIP)\",\"Private Insurance\"],\"answer\":\"Private Insurance\"},{\"number\":3,\"text\":\"During the past year, what was the total combined income for you and the family members you live with? This information will help us determine if you are eligible for any benefits\",\"answers\":[\"Under $20,000\",\"Between $20,000 and $40,000\",\"Between $40,000 and $60,000\",\"Over $60,000\",\"I choose not to answer this question\"],\"answer\":\"Between $40,000 and $60,000\"}]'),
```

Comment:
I think `This information will help us determine if you are eligible for any benefits` is missing a period in each text block.

Reply:
Yes, in general the requirements were not consistent in where to put periods, etc. Since this is for demo purposes only, I'm not going to be a stickler. Thanks!